README.md

```diff
@@ -128,16 +129,14 @@ To get a local copy up and running, follow these steps.
 
 In order to run this project you need:
 
- gem install rails
-
+gem install rails
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-  cd my-folder
-  git clone git@github.com:https://github.com/Theodoraldo/Hello-Microverse
-
+cd my-folder
+git clone git@github.com:https://github.com/Theodoraldo/Hello-Microverse
```

Comment:
- [ ] Kindly, make sure you provided the true link in this section to make your README more professional. 👍